### PR TITLE
Minor updates to getting-started-tutorial

### PR DESCRIPTION
**Pull request by gilad-shaham**

### CODE REVIEW

File: getting-started-tutorial/getting-started-tutorial.ipynb [hunk 1]
```diff
@@ -1556,8 +1549,49 @@
...
-    "project.set_function('hub://describe', 'describe')\n",
-    "describe = project.func('describe').apply(mount_v3io())"
+    "describe = project.set_function('hub://describe', 'describe').apply(mount_v3io())"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### View the function documentation"
```
Comment (Sharon-iguazio) [inline getting-started-tutorial/getting-started-tutorial.ipynb:1559]: Our convention is to use title case (not sentence case) in notebook headings (it's currently not consistent across all NBs, but it's applied in this NB).

File: getting-started-tutorial/getting-started-tutorial.ipynb [hunk 2]
```diff
@@ -1556,8 +1549,49 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "View the function description and parameters by calling the `doc` method:"
```
Comment (Sharon-iguazio) [inline getting-started-tutorial/getting-started-tutorial.ipynb:1566]: I would rephrase:
```markdown
Use the `doc` method to view the embedded documentation of the `describe` function.
```

File: getting-started-tutorial/getting-started-tutorial.ipynb [hunk 3]
```diff
@@ -1556,8 +1549,49 @@
...
+      "default handler: summarize\n",
+      "entry points:\n",
+      "  summarize: Summarize a table\n",
+      "    context(MLClientCtx)  - the function context, default=\n",
+      "    table(DataItem)  - MLRun input pointing to pandas dataframe (csv/parquet file path), default=\n",
+      "    label_column(str)  - ground truth column label, default=None\n",
+      "    class_labels(List[str])  - label for each class in tables and plots, default=[]\n",
+      "    plot_hist(bool)  - (True) set this to False for large tables, default=True\n",
+      "    plots_dest(str)  - destination folder of summary plots (relative to artifact_path), default=plots\n",
+      "    update_dataset  - when the table is a registered dataset update the charts in-place, default=False\n"
```
Comment (Sharon-iguazio) [inline getting-started-tutorial/getting-started-tutorial.ipynb:1589]: I hadn't reviewed this embedded MLRun API doc yet, and I don't like it. It doesn't fit the conventions that I had already started applying when I worked on the MLRun docs (such as beginning the descriptions with a capital letter) and there are grammar issues, etc. Perhaps we can omit this output until a later MLRun/MRLun Functions version with updated doc? We can try to prioritize the doc review for the `describe` function.

Comment (gilad-shaham) [reply getting-started-tutorial/getting-started-tutorial.ipynb:1589]: I think that if we need to work with what we have. I'd like people to know that .doc() exists and that they can use that. Once we get to address the description of the marketplace functions, it will be fixed for all users automatically.

File: getting-started-tutorial/getting-started-tutorial.ipynb (hunk 1 above)
Comment (gilad-shaham) [reply getting-started-tutorial/getting-started-tutorial.ipynb]: Fixed in latest commit

File: getting-started-tutorial/getting-started-tutorial.ipynb (hunk 2 above)
Comment (gilad-shaham) [reply getting-started-tutorial/getting-started-tutorial.ipynb]: fixed in latest commit
